int32 waypointNumber
---
string completed
